### PR TITLE
Fixed some eslint errors

### PR DESCRIPTION
**Pull request by zachdixon**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
Fixes some eslint errors

## Related Tickets & Documents
#3777 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
NA

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Refactor

### CODE REVIEW

File: app/assets/javascripts/initializers/initializeAllFollowButts.js [hunk 1]
```diff
@@ -107,26 +83,29 @@ function handleOptimisticButtRender(butt) {
...
+              }
             }
+          } catch (err) {
+            return err;
           }
-        } catch (err) {
-          return;
-        }
-      });
+          return null;
```
Comment (rhymes) [inline app/assets/javascripts/initializers/initializeAllFollowButts.js]: why `return null`? apparently no one is checking the return value of this function

File: app/assets/javascripts/initializers/initializeArticleReactions.js [hunk 2]
```diff
@@ -36,62 +38,32 @@ function hasUserReacted(reactionName) {
 function getNumReactions(reactionName) {
   var num = document.getElementById('reaction-number-' + reactionName)
     .textContent;
-  if (num == '') {
+  if (num === '') {
     return 0;
   }
-  return parseInt(num);
+  return parseInt(num, 10);
```
Comment (rhymes) [inline app/assets/javascripts/initializers/initializeArticleReactions.js]: yes! :)

File: app/assets/javascripts/initializers/initializeArticleReactions.js [hunk 3]
```diff
@@ -109,14 +81,13 @@ function reactToArticle(articleId, reaction) {
...
-  if (userStatus == 'logged-out') {
+  window.sendHapticMessage('medium');
+  if (userStatus === 'logged-out') {
     showModal('react-to-article');
     return;
-  } else {
-    toggleReaction();
-    document.getElementById('reaction-butt-' + reaction).disabled = true;
   }
+  toggleReaction();
```
Comment (rhymes) [inline app/assets/javascripts/initializers/initializeArticleReactions.js]: why not remove the `return` statement? just curious :D

File: app/assets/javascripts/initializers/initializeArticleReactions.js [hunk 4]
```diff
@@ -132,39 +103,74 @@ function reactToArticle(articleId, reaction) {
 
   getCsrfToken()
     .then(sendFetch('reaction-creation', createFormdata()))
-    .then(function(response) {
+    .then(function getCsrfTokenCallback(response) {
```
Comment (rhymes) [inline app/assets/javascripts/initializers/initializeArticleReactions.js]: you can switch to arrow functions if you don't want to invent names for inline callbacks and errbacks

File: app/assets/javascripts/initializers/initializeSwipeGestures.js [hunk 5]
```diff
@@ -1,61 +1,93 @@
...
+    swipeState = 'middle';
+    slideSidebar('right', 'outOfView');
   }
 }
+
+function initializeSwipeGestures() {
+  initializeSwipeGestures.called = true;
+  swipeState = 'middle';
+  setTimeout(function swipeGesturesTimeout() {
+    // window.onload=function(){
```
Comment (rhymes) [inline app/assets/javascripts/initializers/initializeSwipeGestures.js]: can be removed...

File: app/assets/javascripts/initializers/initializeSwipeGestures.js [hunk 6]
```diff
@@ -1,61 +1,93 @@
-function initializeSwipeGestures(){
-  initializeSwipeGestures.called = true;
-  swipeState = "middle";
-  setTimeout(function(){
+'use strict';
```
Comment (rhymes) [inline app/assets/javascripts/initializers/initializeSwipeGestures.js]: this file unfortunately has cryptic variable names and no comments :( (outside of the scope of this PR)

**Comment by zachdixon** [comment]
Hey @rhymes , thanks for checking out my PR, apologies for not commenting in #3777 and also for not getting back sooner. I'll take a look at updating this PR with your suggestions as soon as I can.

**Comment by mstruve** [comment]
Hey @zachdixon! We are in the process of going through and cleaning up some of our PR history. Since this PR has not been active for a while, has some conflicts, and is not super critical for the app functionality I am going to close it for now. 

If you would still like to get this merged please feel free to resolve the conflicts and then open the PR back up. We would be more than happy to help push this through if you would like!

Thanks so much for being an awesome contributor!

**Pull request closed without merging**